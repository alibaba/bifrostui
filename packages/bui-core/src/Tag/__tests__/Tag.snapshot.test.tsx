import { snapshotTest } from 'testing';

describe('Tag snapshot', () => {
  snapshotTest('Tag');
});
