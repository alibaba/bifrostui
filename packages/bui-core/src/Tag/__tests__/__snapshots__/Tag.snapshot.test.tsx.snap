// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag snapshot Tag demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-tag bui-tag-outlined bui-tag-default"
    style={Object {}}
  >
    默认标签
  </div>
  <div
    className="bui-tag bui-tag-contained bui-tag-default"
    style={Object {}}
  >
    填充标签
  </div>
  <div
    className="bui-tag bui-tag-light bui-tag-default"
    style={Object {}}
  >
    浅亮标签
  </div>
</div>
`;

exports[`Tag snapshot Tag demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "10px",
      }
    }
  >
    <div
      className="bui-tag bui-tag-outlined bui-tag-primary"
      style={Object {}}
    >
      限时特价
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-info"
      style={Object {}}
    >
      音乐剧
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-success"
      style={Object {}}
    >
      音乐剧
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-warning"
      style={Object {}}
    >
      点映
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-danger"
      style={Object {}}
    >
      限时优惠
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-vip"
      style={Object {}}
    >
      会员
    </div>
  </div>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "10px",
      }
    }
  >
    <div
      className="bui-tag bui-tag-contained bui-tag-primary"
      style={Object {}}
    >
      早鸟优惠
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-info"
      style={Object {}}
    >
      音乐剧
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-success"
      style={Object {}}
    >
      影城卡
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-warning"
      style={Object {}}
    >
      点映
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-danger"
      style={Object {}}
    >
      限时优惠
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-vip"
      style={Object {}}
    >
      会员
    </div>
  </div>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "row",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "10px",
      }
    }
  >
    <div
      className="bui-tag bui-tag-light bui-tag-primary"
      style={Object {}}
    >
      早鸟优惠
    </div>
    <div
      className="bui-tag bui-tag-light bui-tag-info"
      style={Object {}}
    >
      音乐剧
    </div>
    <div
      className="bui-tag bui-tag-light bui-tag-success"
      style={Object {}}
    >
      影城卡
    </div>
    <div
      className="bui-tag bui-tag-light bui-tag-warning"
      style={Object {}}
    >
      点映
    </div>
    <div
      className="bui-tag bui-tag-light bui-tag-danger"
      style={Object {}}
    >
      限时优惠
    </div>
    <div
      className="bui-tag bui-tag-light bui-tag-vip"
      style={Object {}}
    >
      会员
    </div>
  </div>
</div>
`;

exports[`Tag snapshot Tag demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-tag bui-tag-contained bui-tag-default"
    style={
      Object {
        "backgroundColor": "#69eeff",
      }
    }
  >
    自定义色彩
  </div>
  <div
    className="bui-tag bui-tag-light bui-tag-default"
    style={
      Object {
        "backgroundColor": "rgba(105,238,255,0.3)",
        "color": "#69eeff",
      }
    }
  >
    自定义色彩
  </div>
  <div
    className="bui-tag bui-tag-outlined bui-tag-default"
    style={
      Object {
        "borderColor": "rgba(105,238,255,0.4)",
        "color": "#69eeff",
      }
    }
  >
    自定义色彩
  </div>
</div>
`;

exports[`Tag snapshot Tag demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-tag-group"
  >
    <div
      className="bui-tag bui-tag-contained bui-tag-primary"
      style={Object {}}
    >
      primary
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-info"
      style={Object {}}
    >
      info
    </div>
  </div>
  <div
    className="bui-tag-group"
  >
    <div
      className="bui-tag bui-tag-contained bui-tag-success"
      style={Object {}}
    >
      success
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-warning"
      style={Object {}}
    >
      warning
    </div>
  </div>
  <div
    className="bui-tag-group"
  >
    <div
      className="bui-tag bui-tag-outlined bui-tag-warning"
      style={Object {}}
    >
      warning
    </div>
    <div
      className="bui-tag bui-tag-contained bui-tag-success"
      style={Object {}}
    >
      success
    </div>
  </div>
  <div
    className="bui-tag-group"
  >
    <div
      className="bui-tag bui-tag-outlined bui-tag-primary"
      style={Object {}}
    >
      primary
    </div>
    <div
      className="bui-tag bui-tag-outlined bui-tag-info"
      style={Object {}}
    >
      info
    </div>
  </div>
</div>
`;
